apps/web/server.js

```diff
@@ -0,0 +1,23 @@
+const { createServer } = require("https");
+const { parse } = require("url");
+const next = require("next");
+const fs = require("fs");
+const port = 3000;
+const dev = process.env.NODE_ENV !== "production";
+const app = next({ dev });
+const handle = app.getRequestHandler();
+
+const httpsOptions = {
+  key: fs.readFileSync("./app.cal.local-key.pem"),
+  cert: fs.readFileSync("./app.cal.local.pem"),
+};
+
+app.prepare().then(() => {
+  createServer(httpsOptions, async (req, res) => {
+    const parsedUrl = parse(req.url, true);
+    await handle(req, res, parsedUrl);
+  }).listen(port, (err) => {
+    if (err) throw err;
+    console.log(`ready - started server on url: https://app.cal.local:${port}`);
+  });
+});
```

Comment:
This script is for serving nextjs server over https for testing organization's cal video meeting links